exec/java-exec/src/test/java/org/apache/drill/exec/planner/logical/TestConvertCountToDirectScan.java

```diff
@@ -238,4 +238,40 @@ public void testCountsWithMetadataCacheSummaryAndDirPruning() throws Exception {
       test("drop table if exists %s", tableName);
     }
   }
+
+  @Test
+  public void testCountsWithWildCard() throws Exception {
+    test("use dfs.tmp");
+    String tableName = "parquet_table_counts";
+
+    try {
+      for (int i = 0; i < 10; i++) {
+        test(String.format("create table `%s/12/%s` as select * from cp.`tpch/orders.parquet`", tableName, i));
```

Comment:
Since the test does not actually need a larger table, could you perhaps use a smaller table (like nation) here since there are 13 CTAS statements and doing a CTAS on 'orders' adds extra time to the test.